prow/flagutil/github.go

```diff
@@ -112,9 +117,17 @@ func (o *GitHubOptions) GitHubClient(secretAgent *secret.Agent, dryRun bool) (cl
 	return o.GitHubClientWithLogFields(secretAgent, dryRun, logrus.Fields{})
 }
 
+// GitHubClientWithAccessToken creates a GitHub client from an access token.
+func (o *GitHubOptions) GitHubClientWithAccessToken(token string) github.Client {
+	noopCensor := func(content []byte) []byte {
+		return content
```

Comment:
We should elide the token here, not no-op.

Reply:
Yeah, I copied the code from its original place. Let me fix this.